src/controllers/articles.js

```diff
@@ -0,0 +1,5 @@
+exports.get = (req, res) => {
+  res.render('articles', {
+    title: 'Articles ', headerFound: true, footerFound: true, style: ['articles', 'header', 'footer'], javascript: ['hambruger'],
```

Comment:
i think you need article.js file , and you don't import it in javascript array.

Reply:
I still didn't use it so I didn't require it here , when I need it I would put it :smiley_cat: